R/read.R

```diff
@@ -7,14 +7,25 @@ st_read_agnostic <- function(dir, start_year, sectors) {
     capacity_factors_power <- NULL
   }
 
+  if (risktype == "trisk") {
   out <- list(
     capacity_factors_power = capacity_factors_power,
     df_price = read_price_data(price_data_file(dir)),
     scenario_data = read_scenario_data(scenario_data_file(dir, start_year)),
     financial_data = read_financial_data(financial_data_file(dir)),
     production_data = read_production_data(production_data_file(dir)),
     carbon_data = read_carbon_data(carbon_price_data_file(dir))
```

Comment:
I would suggest to make the if else condition only around carbob_data and also add a comment why this is happening. This would make the difference much easier to spot and understand

Reply:
ok done - hopefully at least :)